src/QuickMarcEditor/QuickMarcEditorRows/LeaderField/LeaderField.js

```diff
@@ -7,15 +7,18 @@ import {
 } from '../BytesField';
 import { leaderConfig } from './leaderConfig';
 import { MARC_TYPES } from '../../../common/constants';
+import { QUICK_MARC_ACTIONS } from '../../constants';
 
 const LeaderField = ({
   name,
   marcType,
   leaderField,
+  action,
 }) => {
   const intl = useIntl();
   const fields = leaderConfig[marcType].map(config => {
     const { allowedValues, name: boxName } = config;
+    const required = marcType === MARC_TYPES.BIB && action === QUICK_MARC_ACTIONS.CREATE && ['Type', 'BLvl'].includes(boxName);
```

Comment:
I think if we also show the asterisk when editing Bib records, then we can add the `required` property to `leaderConfig` constant. Let's ask Khalilah about this.
If she thinks we shouldn't show it for edit view, then we can still update the config, but return `required` from this function conditionally - if action is Create.